api/middleware/responseLogger.go

```diff
@@ -0,0 +1,104 @@
+package middleware
```

Comment:
tests are missing for this new functionality

Reply:
added